src/auxinfo/info.rs

```diff
@@ -41,6 +50,78 @@ impl AuxInfoPrivate {
     pub(crate) fn decryption_key(&self) -> &DecryptionKey {
         &self.decryption_key
     }
+
+    /// Convert private material into bytes.
+    ///
+    /// 🔒 This is intended for use by the calling application for secure
+    /// storage. The output of this function should be handled with care.
+    pub fn into_bytes(self) -> Vec<u8> {
+        // Format:
+        // AUXINFO_TAG | key_len in bytes | key
+        //             | ---8 bytes------ | --key_len bytes---
+
+        let key = self.decryption_key.into_bytes();
+        let key_len = key.len().to_le_bytes();
+
+        [AUXINFO_TAG, &key_len, &key].concat()
```

Comment:
(Same as other PR). Should `key` be zeroized?

Reply:
Yes! Also fixed the other matching zeroize issue in parsing.